app/views/people/_details_sjas.html.haml

```diff
@@ -0,0 +1,12 @@
+-#  Copyright (c) 2012-2020, Stiftung für junge Auslandssschweizer. This file is part of
+-#  hitobito_sjas and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_sjas.
+
+= render_attrs(entry, :place_of_origin)
+
+- language_skills = [:language_skill_de, :language_skill_fr, :language_skill_it,
+                     :language_skill_en, :language_skill_es]
+- if entry.other_native_language.present?
+  - language_skills << :other_native_language
+= render_attrs(entry, *language_skills)
```

Comment:
Mir ist nicht ganz klar wie ihr da leere Werte handhabt. Habe nun mal das Attribut "Andere Muttersprache" ganz ausgeblendet, wenn es nicht gesetzt ist, da es sich hier um eine Zusatzinfo handelt, die wahrscheinlich nur wenige ausgefüllt haben.

Reply:
in diesem Fall würde ich mit `render_present_attrs` einfach nur die anzeigen, die schon ausgeüllt sind. So sieht man nur das, wozu die Person sich schon geäußert hat. Im Falle der "anderen Muttersprache" kann man das angezeigte Feld wieder entfernen, bei den anderen. Das finde ich in dem Kontext noch recht praktisch.